### PR TITLE
Add dns0.eu

### PR DESCRIPTION
**Pull request by NixCookies**
## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] Documentation update
- [ ] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
Added dns0.eu, a European public DNS service.
https://www.dns0.eu/

## Testing
- Successfully compiled using Compile.ps1 without issues.
- Ran winutil.ps1 and was able to select and change the DNS settings.
- Verified the settings in Windows.
- Tested on Windows 11 IoT Enterprise LTSC 24H2.

## Impact
- Additional DNS Options: This change adds more flexibility for users by providing an additional public DNS service.
- Regional Benefits: Specifically beneficial for users in European regions, offering a geographically closer DNS service.

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Last Updated: 2024-08-07
   "panel": "1",
   "Order": "a040_",
   "Type": "Combobox",
-  "ComboItems": "Default DHCP Google Cloudflare Cloudflare_Malware Cloudflare_Malware_Adult Open_DNS Quad9 AdGuard_Ads_Trackers AdGuard_Ads_Trackers_Malware_Adult",
+  "ComboItems": "Default DHCP Google Cloudflare Cloudflare_Malware Cloudflare_Malware_Adult Open_DNS Quad9 AdGuard_Ads_Trackers AdGuard_Ads_Trackers_Malware_Adult dns0.eu_Open dns0.eu_ZERO dns0.eu_KIDS",
```
Comment (Real-MullaC) [inline docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md:24]: Do not edit this section of the code. This is auto generated.

**Review by Real-MullaC — CHANGES_REQUESTED** [review]
Please remove this bit.

File: docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md
Comment (ruxunderscore) [inline docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md:1]: @NixCookies As Mulla said. You should be able to remove the revert the changes on this file, and do a new commit on your side. Once you push the changes it should update on the PR. One of us can help in the Discord, if you don't know how that's done.

File: docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md (hunk 1 above)
Comment (NixCookies) [reply docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md]: Done

**Review by ruxunderscore — CHANGES_REQUESTED** [review]
As mentioned by Mulla, we just need changes to changedns.md removed, since this is an automatically generated file.

File: docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md
Comment (NixCookies) [reply docs/dev/tweaks/z--Advanced-Tweaks---CAUTION/changedns.md:1]: I think i did it

**Review by ruxunderscore — APPROVED** [review]

**Review by Real-MullaC — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]
